src/resources/UsageAnalytics/Read/DataHealth/DataHealthInterfaces.ts

```diff
@@ -0,0 +1,350 @@
+import {Paginated} from '../../../BaseInterfaces.js';
+
+/**
+ * Pagination information about which page is included in the response,
+ * and what the full range of data is.
+ */
+export interface PaginationInformationModel {
+    /**
+     * Zero-based page index of the current page.
+     */
+    page: number;
+    /**
+     * The number of items for each page.
+     * Note that the actual number of items may be lower for the last page.
+     */
+    perPage: number;
+    /**
+     * Total number of items available (across all pages).
+     */
+    totalItems: number;
+    /**
+     * The number of pages available, based on `totalItems` and `perPage`.
+     */
+    totalPages: number;
+}
```

Comment:
I think we should add `PaginationInformationModel` to `BaseInterfaces` instead of here, as it is completely generic. (It applies to each paginated response that follows the [api-guidelines](/coveo/api-guidelines/blob/main/Guidelines.md#42-paginated-api-call---response)).